.github/CODEOWNERS

```diff
@@ -555,13 +555,15 @@
 /core/templates/services/autoplayed-videos.service*.ts @aks681
 /core/templates/services/external-save.service*.ts @srijanreddy98
 /core/templates/services/external-rte-save.service*.ts @srijanreddy98
+/core/templates/services/oppia-rte-parser.service.ts @srijanreddy98 @aks681
```

Comment:
I am planning to add more tests to these files later... until those tests have been added, I will be sharing the ownership of the two files added.